### PR TITLE
8262066: ProblemList java/util/Locale/LocaleProvidersRun.java

### PR DESCRIPTION
**Pull request by naotoj**
The subject test case is failing under the JMS-enabled environment.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [ ] Change must be properly reviewed

### Issue
 * [JDK-8262066](https://bugs.openjdk.java.net/browse/JDK-8262066): ProblemList java/util/Locale/LocaleProvidersRun.java

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/2657/head:pull/2657`
`$ git checkout pull/2657`

### CODE REVIEW

File: test/jdk/ProblemList.txt [hunk 1]
```diff
@@ -796,6 +796,8 @@ com/sun/jdi/AfterThreadDeathTest.java                           8232839 linux-al
 
 # jdk_util
 
+java/util/Locale/LocaleProvidersRun.java                        8261919 generic-all
```
Comment (dcubed-ojdk) [inline test/jdk/ProblemList.txt:799]: This only needs to be ProblemListed on linux-x64.

**Review by dcubed-ojdk — CHANGES_REQUESTED** [review]

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (naotoj) [reply test/jdk/ProblemList.txt:799]: It's not platform specific, but specific to the environment that is usage tracker enabled. Thus I put it as 'generic'.

**Comment by naotoj** [comment]
As the original issue was resolved, no need to problem list this.

**Pull request closed without merging**